FrameRate/FrameRate.cpp

```diff
@@ -407,7 +406,7 @@ namespace WPEFramework
             {
                 m_minFpsValue = newFpsValue;
             }
-            m_totalFpsValues += newFpsValue;
+	    m_totalFpsValues = m_maxFpsValue + m_minFpsValue;
```

Comment:
Why is this changed to m_maxFpsValue + m_minFpsValue? The m_totalFpsValues is supposed to track the fps updates.